lib/ansible/runner.rb

```diff
@@ -79,6 +79,18 @@ def run(env_vars, extra_vars, playbook_path, tags: nil, hosts: ["localhost"], cr
                     :become_enabled => become_enabled)
       end
 
+      def run_vault(env_vars, extra_vars, playbook_path, tags: nil, hosts: ["localhost"], credentials: [], verbosity: 0, become_enabled: false)
+        run_via_cli(hosts,
+                    credentials,
+                    env_vars,
+                    extra_vars,
+                    :tags           => tags,
+                    :playbook       => playbook_path,
+                    :test_vault     => true,
```

Comment:
This doesn't feel right for us to add a new public API just for testing.  Is there a reason you couldn't use the regular API with a credential?